apps/website/components/organisms/data-grid/data-grid.tsx

```diff
@@ -223,7 +223,7 @@ const defineCols = (columns: IColumnGrid[]) => {
                 textOverflow: 'ellipsis',
               }}
             >
-              {params?.issuerUser?.gatewayId}
+              {params.issuerUser.gatewayId}
```

Comment:
why did you removed the optional chaining? if you remove it, it can lead to error on the component, if its necessary, create a fallback
```suggestion
              {params?.issuerUser?.gatewayId ?? FALLBACK}
```